notepad.sql

```diff
@@ -1,6 +1,7 @@
+CREATE DATABASE API_notes;
+
 CREATE TABLE notes (
```

Comment:
Why do you use a plural name for your table?

Reply:
because, i want to get multiple notes in my table